src/pages/archives/404.tsx

```diff
@@ -0,0 +1,32 @@
+import { Location } from "@reach/router"
+import { graphql } from "gatsby"
+import React, { Fragment } from "react"
+
+import { FileConnection } from "../../../generated/graphql"
+import { ComponentQuery } from "../../../typings"
+import { SEO } from "../../components/SEO"
+
+import { getPossibleCorrections } from "../../components/404LinkCorrection"
+
+export default ({ data }: ComponentQuery<{ allFile: FileConnection }>) => (
+    <Fragment>
+        <SEO title="404: Archive Not found" />
+        <h1>RESOURCE NOT FOUND</h1>
+        <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
+        <Location>
+            {({ location }) => getPossibleCorrections("archives/", location, data, 8)}
```

Comment:
can we default to 8 in the function? :)

Reply:
I was thinking about it, but I was also thinking we may need to change it for tech spotlights?

Reply:
Yes maybe! But right now I would be okay to default to 8 and get rid of the verbosity here